docs/sql/scalar.txt

```diff
@@ -835,9 +835,152 @@ Examples
     +---------------------+--------------+
     SELECT 5 rows in set (... sec)
 
+
+Array Functions
+===============
+
+array_cat(first_array, second_array, [ third_array , ... ]) returns array
```

Comment:
after long internal discussion we decided, it would be nice to support the concat operator for arrays ``||`` as well.
It is currently only implemented as concatenating strings and other types to one string: https://github.com/crate/crate/blob/master/sql/src/main/java/io/crate/operation/scalar/ConcatFunction.java

Please consider resolving the operator to your function there.